integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/util/SecondaryIndexUtil.scala

```diff
@@ -353,16 +342,38 @@ object SecondaryIndexUtil {
     }
   }
 
+  /**
+   * This method returns the list of index/merge index files for a segment in carbonTable.
+   */
+  @throws[IOException]
+  private def getIndexFilesListForSegment(segment: Segment, tablePath: String): util.Set[String] = {
+    var indexFiles : util.Set[String] = new util.HashSet[String]
+    val segmentFileStore = new SegmentFileStore(tablePath,
+      segment.getSegmentFileName)
+    val segmentPath = CarbonTablePath.getSegmentPath(tablePath, segment.getSegmentNo)
+    if (segmentFileStore.getSegmentFile == null)  {
+      indexFiles = new SegmentIndexFileStore()
+        .getMergeOrIndexFilesFromSegment(segmentPath).keySet
+    }
+    else {
```

Comment:
move this to above line, correct the style

Reply:
Done